config/vufind/ComposedDriver.ini

```diff
@@ -0,0 +1,36 @@
+[General]
+; Default driver for all methods
+;main_driver = "driver1"
+
+; (Optional) the path to the driver configurations relative to VuFind config
+; dir -- omit to locate those in VuFind config dir -- it also could be an
+; absolute path
+;drivers_config_path = private
+
+; Definition of all the used drivers in the form <DriverName> = <DriverClass>
+; The name of config file for a driver needs te be the same <DriverName>
+[Drivers]
+;driver1 = PAIA
+;driver2 = DAIA
+;driver3 = Demo
+
+; For each driver method it can be specified which drivers should be used and how they can be composed.
+; The section needs to be [<driver method name>]. main_driver specifies the driver that is primarily used.
+; The results of the main driver can be composed with other driver results. Therefore, one needs to specify
+; merge_keys for the support drivers. All results of the support drivers are then matched with the keys of the main
+; driver. E.g. for the method getHolding the results of multiple drivers can get composed if they match on "item_id".
+; The array support_keys specifies which of there keys of the given support driver should be composed with the main result.
+; The format is: support_keys["<DriverName"] = "<key1>,<key2>,...,<keyN>"
+
+; Here are some examples how a configuration could look like for a combination of DAIA and PAIA
+; but it most likely wont work with the Demo driver
```

Comment:
"examples of how" might read a little better. "wont" should be "won't". I'm not entirely sure I understand why the Demo driver won't work, or why that is mentioned here (driver2 is not set to Demo in the example above).

Reply:
fixed. I also explained why the Demo driver won't work.